scripts/install-info.sh

```diff
@@ -0,0 +1,139 @@
+#!/bin/bash
+
+# Constants
+GITHUB_DOCS_URL='https://pipelinesascode.com/docs/install/github_apps/'
+DEFAULT_RHDH_DEPLOYMENT="backstage-ai-rh-developer-hub" # deployment created by rhdh operator by default
+EXTRA_ENV_SECRET="ai-rh-developer-hub-env" # secret created by rhdh installer to store private env vars
+
+# Variables
+BASE_DIR="$(realpath $(dirname ${BASH_SOURCE[0]}))"
+RHDH_DEPLOYMENT="${DEFAULT_RHDH_DEPLOYMENT}"
+RHDH_ROUTE="${RHDH_ROUTE:-$RHDH_DEPLOYMENT}"
```

Comment:
Nit: Do we need this intermediate var?

```suggestion
RHDH_ROUTE="${RHDH_ROUTE:-$DEFAULT_RHDH_DEPLOYMENT}"
```

Reply:
Also, am I missing where this is set to something other than the default for those who that use case applies to?

Reply:
> Nit: Do we need this intermediate var?

@Jdubrick `RHDH_ROUTE="${RHDH_ROUTE:-$RHDH_DEPLOYMENT}"` is correct, if the RHDH deployment name is set to something else, the route name should be too.